partcad/src/partcad/part_factory_brep.py

```
@@ -0,0 +1,33 @@
+from OCP.BRep import BRep_Builder
+from OCP.BRepTools import BRepTools
+from OCP.TopoDS import TopoDS_Shape
+import os
```

Ruff (0.8.2)

`os` imported but unused

Remove unused import: `os`

(F401)